app/views/queries/index.html.erb

```diff
@@ -13,6 +13,14 @@
       <th>Best bets</th>
       <th>Worst bets</th>
     </tr>
+
+    <tr class="table-header">
```

Comment:
I see. Would it be useful to add table helper to https://github.com/alphagov/govuk_admin_template to enforce the correct markup of tables?

Something like Active Admin's `table_for`? http://activeadmin.info/docs/12-arbre-components.html
 
```ruby
table_for order.payments do
  column(:payment_type) { |payment| payment.payment_type.titleize }
  column "Received On",     :created_at
  column "Details & Notes", :payment_details
  column "Amount",          :amount_in_dollars
end
```